python/hail/dataset.py

```diff
@@ -3238,18 +3241,24 @@ def lmmreg(self, kinshipMatrix, y, covariates=[], global_root="global.lmmreg", v
 
         **Kinship Matrix**
 
-        FastLMM uses the Realized Relationship Matrix (RRM) for kinship. This can be computed with :py:meth:`~hail.VariantDataset.rrm`. However, any instance of :py:class:`KinshipMatrix` may be used, so long as ``sample_list`` contains the complete samples of the caller variant dataset in the same order.
+        FastLMM uses the Realized Relationship Matrix (RRM) for kinship. This can be computed with :py:meth:`~hail.VariantDataset.rrm`. However, any instance of :py:class:`KinshipMatrix` may be used, so long as ``sample_list`` contains the complete samples of the caller variant dataset in the same order. Currently, due to the fact that the eigendecomposition is done locally, this matrix (or the LD matrix below) can only be a maximum size of 32,000 X 32,000.
```

Comment:
FastLMM uses the Realized Relationship Matrix (RRM) for kinship, which may be computed with :py:meth:`~hail.VariantDataset.rrm`. However, any instance of :py:class:`KinshipMatrix` may be used in :py:meth:`~hail.VariantDataset.lmmreg`, so long as ...